src/id/onepieceberwarna/src/eu/kanade/tachiyomi/extension/id/onepieceberwarna/OnePieceBerwarna.kt

```diff
@@ -0,0 +1,143 @@
+package eu.kanade.tachiyomi.extension.id.onepieceberwarna
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import rx.Observable
+import rx.Single
+
+class OnePieceBerwarna : ParsedHttpSource() {
+
+    override val name = "One Piece Berwarna"
+
+    override val baseUrl = "https://onepieceberwarna.com"
+
+    override val lang = "id"
+
+    override val supportsLatest = false
+
+    // one page contains all chapters
+    override fun popularMangaRequest(page: Int): Request = GET(baseUrl, headers)
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        val mainElements = response.asJsoup()
+        val list = mutableListOf<SManga>()
+        var saga: String? = null
+        var image: String?
+        var volume: String? = null
+
+        val sections = mainElements.select("section:not(:first-child)")
+
+        run loop@{
+            sections.forEach { element ->
+                val dataID = element.attr("data-id")
+                if (dataID == "49afe94") {
+                    // break on non-manga section
+                    return@loop
+                }
+
+                element.select(".elementor-col-100:not(:has(div[data-element_type=icon.default])) h1")
+                    .first()?.let {
+                        saga = it.text()
+                    }
+
+                element.select(".elementor-col-50 h1").first()?.let {
+                    volume = it.text()
+                }
+
+                element.select(".elementor-widget-container > img").let {
```

Comment:
```suggestion
                element.select(".elementor-widget-container > img[data-src]").let {
```
ini juga kenapa ga begini aja daripada ngecek pake IF